### ISSUE TITLE
[QA] [Win] MSI installer: certificate expire date < 1 year?

### ISSUE THREAD

**Issue by gabi18** [owncloud/client#8591]
Please check the certificate expire date in MSI installer. Last time I saw it, the date was 15.02.2022. 

I am not sure about the current setting because I cannot get the MSI installer window for a fresh installation.

**Comment by TheOneRing** [comment]
![image](https://user-images.githubusercontent.com/200626/116854891-b1e8df80-abf8-11eb-9c1b-c520a4c0fca2.png)

Not an issue, but we should refresh the certificate before it runs out.
Windows will not warn for applications signed with an outdated cert, but from then on we won't be able to sign new files.

**Comment by gabi18** [comment]
Meanwhile I have found where the certificate can be checked - I expected it at a different step during  installation...

**Comment by jnweiger** [comment]
![image](https://user-images.githubusercontent.com/1108546/116865201-799dcd00-ac09-11eb-911e-9ad06e88f973.png)


Whooo, With RC1, I get one day less than what @gabi18 got :scream:

**Comment by TheOneRing** [comment]
Different time zone? It also displays another valid from date.
Anyhow not an issue :P

**Comment by jnweiger** [comment]
If it is not an issue, should we remove the cert-check from the testplan?
Unless removed or clarified, we'll continue to report expiry dates of less than 1y.

**Comment by jnweiger** [comment]
Seen still with 2.9.0-rc2.5134

![image](https://user-images.githubusercontent.com/1108546/132336385-f6d25348-3d23-41ca-b6f0-2902b5def2ae.png)

**Comment by TheOneRing** [comment]
Yes so we need a new one once we plan to release after valentines day ❤️

**Comment by gabi18** [comment]
Also valid for 2.10.0, expiry date is 15.02.2022

![win-certificate](https://user-images.githubusercontent.com/75983861/148758723-7f7c4232-e3fe-4847-9d7d-d027e1ae387f.png)
